### PR TITLE
feat: Add @CODINATA as a contributor

### LINKED ISSUE #329: Feature: Add @CODINATA as a contributor.
### Description

As part of the Intro to Open Source course, we are encouraged to contribute to this repo.

### Suggested solution

Add myself as a contributor.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by CODINATA**
## Description
This PR adds @CODINATA  as a contributor and it is done for practicing purposes  to get familiar with open source contributions and guidelines.

## What type of PR is this? (check all applicable)

- [X] 🤝 Add a contributor
- [X] 📝 Documentation Update

## Related Issues

Closes #329 

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [X] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [X] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [X] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)
opensource

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @CODINATA,

Thank you for your contribution! ✨ 

We have merged a PR before yours. Please **[resolve the conflicts](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository)** before we can review yours.

Let us know if you have questions or need any help. 😊

**Comment by adiati98** [comment]
@CODINATA, you haven't run the generate command after resolving the conflicts.

File: .all-contributorsrc [hunk 1]
```diff
@@ -2134,9 +2135,22 @@
...
+{
+ 
+      "login": "CODINATA",
+      "name": "Mohamed Adam Ata",
+      "avatar_url": "https://avatars.githubusercontent.com/u/115686926?v=4",
+      "profile": "https://github.com/CODINATA",
+      "contributions": [
+        "question",
+        "tutorial"]
+      }
```
Comment (adiati98) [inline .all-contributorsrc:2156]: Move these to line 2141.

**Review by adiati98 — CHANGES_REQUESTED** [review]
I've left a review here. 🙂
You also haven't run the generate command.

**Review by adiati98 — CHANGES_REQUESTED** [review]
- We don't want to remove these lines:

```markdown
  "contributorsPerLine": 7,
  "linkToUsage": true
```

- You must fix the format in `.allcontributorsrc`. Please look at the [existing one](https://github.com/open-sauced/guestbook/blob/main/.all-contributorsrc) to do that.

**Review by adiati98 — APPROVED** [review]
Hey @CODINATA,

Looking at the changes that you made, you:

- deleted your own data in the `.all-contributorsrc`. When you don't have data, you actually can't have your profile generated on README after you run the command.
- deleted the footer in the README. This **may not be deleted**.

I've helped you fixing the files and run the command. Please take a look at [the commits](https://github.com/open-sauced/guestbook/pull/330/commits) that I made to understand how I did it.

For your future reference:

- Please read the instructions carefully.
- Always ask questions or help when you're in doubt. Communication in open source is very important. Lack of communication can block the progress of review process and your PRs get merged.
- Refrain from asking review from maintainers when you haven't solved all their feedback. 

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

git merge GIF

**Comment by CODINATA** [comment]
thank you @adiati98 for your support I appreciate your work and comprehension

**Pull request merged**